iceberg/iceberg-catalog/src/main/java/org/apache/iceberg/hive/HiveOperationsBase.java

```diff
@@ -56,6 +59,21 @@ public interface HiveOperationsBase {
 
   TableType tableType();
 
+  enum ContentType {
+    TABLE("Table"),
+    VIEW("View");
+
+    private final String value;
+
+    ContentType(String value) {
+      this.value = value;
+    }
+
+    public String value() {
```

Comment:
why do we need string value? also I think HiveCatalog would be a better place for the enum

Reply:
This part is for the coming iceberg native view. Can we optimize this in the next backport?  (integrate the iceberg native view)

Reply:
👍